tools/ui-components/src/colors.css

```diff
@@ -110,10 +120,20 @@ div.dark {
   --default-foreground-secondary: var(--gray05);
   --default-foreground-quaternary: var(--gray15);
   --default-foreground-danger: var(--red90);
+  --default-foreground-info: var(--blue90);
 
   --default-background-primary: var(--gray90);
   --default-background-secondary: var(--gray85);
   --default-background-tertiary: var(--gray80);
   --default-background-quaternary: var(--gray75);
   --default-background-danger: var(--red15);
+  --default-background-info: var(--blue30);
```

Comment:
@ahmadabdolsaheb I already forgot why we went with the "default" prefix, but looking at these now, I wonder if we should remove it to keep the variables name a bit shorter. The "default" part implies that there are overrides, but I don't believe we want that.

Same thoughts on the color names in the tailwind config file.

Reply:
yep, we could drop them.

Reply:
Let me know if you have done the research on best practices; otherwise the secnario above seems plausible and we could batch it in a pr with the `default` removals.

Reply:
Sorry, I have stepped away from the work a bit. 

Your suggestion makes sense now that I'm looking at the variable list again. I think for states (focus, hover, active), we can just use an existing variable instead of having a dedicated set for them, since the states for a button might be different from, say, an `<input />`.

I'll have a PR to clean these up and add a note on this in the contributing doc 🙂